flink-runtime/src/main/java/org/apache/flink/runtime/blob/AbstractBlobCache.java

```diff
@@ -249,6 +278,28 @@ public void close() throws IOException {
         }
     }
 
+    /**
+     * Delete the blob file with the given key.
+     *
+     * @param jobId ID of the job this blob belongs to (or <tt>null</tt> if job-unrelated)
+     * @param blobKey The key of the desired BLOB.
+     * @return file successfully deleted or not.
+     */
+    public boolean deleteFileUnsafely(@Nullable JobID jobId, BlobKey blobKey) {
+        final File localFile =
+                new File(
+                        BlobUtils.getStorageLocationPath(
+                                storageDir.getAbsolutePath(), jobId, blobKey));
+        if (!localFile.delete() && localFile.exists()) {
+            log.warn(
+                    "Failed to delete locally cached BLOB {} at {}",
+                    blobKey,
+                    localFile.getAbsolutePath());
+            return false;
```

Comment:
Why do we have a return value but add no handling for it?

Reply:
> Why do we have a return value but add no handling for it?

Since the failure of deleting files is tolerable and the log has been outputted here, we can remove the return value.